tokio/tests/macros_select.rs

```diff
@@ -1,4 +1,5 @@
 #![allow(clippy::blacklisted_name)]
+#![deny(warnings)]
```

Comment:
```suggestion
```

This should be done by ci:
https://github.com/tokio-rs/tokio/blob/e7091fde786722a5301270e6281fc3c449dcfc14/.github/workflows/ci.yml#L10

Reply:
Good point.

Reply:
btw, I noticed some tests that set another `RUSTFLAGS` are overwriting `-Dwarnings`...
https://github.com/tokio-rs/tokio/blob/e7091fde786722a5301270e6281fc3c449dcfc14/.github/workflows/ci.yml#L94
https://github.com/tokio-rs/tokio/blob/e7091fde786722a5301270e6281fc3c449dcfc14/.github/workflows/ci.yml#L159
https://github.com/tokio-rs/tokio/blob/e7091fde786722a5301270e6281fc3c449dcfc14/.github/workflows/ci.yml#L242